### ISSUE TITLE
`no_implicit_braces` false positive on class instance method after custom property

### ISSUE THREAD

**Issue by aseemk** [clutchski/coffeelint#458]
```coffee
class Foo

    Object.defineProperty @::, 'bar',
        enumerable: true
        get: -> true

    toString: ->
        'foo'
```

The `toString: ->` line yields a `no_implicit_braces` error.

([Real world example](https://github.com/thingdom/node-neo4j/blob/0d20a7e0826668dbf83f731d8b9b82ee71ae2db9/lib-new/Transaction.coffee#L69))

Not sure exactly what the trigger is. If I replace the `Object.defineProperty` call with some other executable code (e.g. `registerFoo()`), it works fine.
